dags/datasets/files_to_datastore/licensed-child-care-centres.yaml

```diff
@@ -0,0 +1,116 @@
+# package name
+licensed-child-care-centres:
+  # DAG info
+  schedule: "@once"
+  dag_owner_name: Yanan
+  dag_owner_email: yanan.zhang@toronto.ca
+
+  # mandatory package attributes
+  title: "Licensed Child Care Centres"
+  date_published: "2019-07-23 17:39:29.065675"
+  refresh_rate: Daily
+  dataset_category: Map
+
+  # optional package attributes
+  owner_division: Children's Services
+  owner_section: null
+  owner_unit: null
+  owner_email: childcare@toronto.ca
+
+  civic_issues: 
+    - affordable_housing
+  topics: 
+    - Community services
+  tags: 
+    - name: childcare
+      vocabulary_id: null
+
+  information_url: https://www.toronto.ca/community-people/children-parenting/children-programs-activities/licensed-child-care/
+  excerpt: Licensed Child Care Centres in the City of Toronto including their capacities by age group.
+  limitations: null
+  notes: |
+    A dataset of all Licensed Child Care Centres in the City of Toronto including their capacities by age group.
+    
+    Projection: Unprojected Lat Long, NAD27 datum
```

Comment:
Get rid of this line please - its old information that we should have gotten rid of a while ago :)